tests/pixels/test_utils.py

```diff
@@ -1768,6 +1979,17 @@ def test_compress_arr(self):
         assert ds.file_meta.TransferSyntaxUID == DeflatedImageFrameCompression
         assert np.array_equal(ds.pixel_array, ref)
 
+    @pytest.mark.skipif(not HAVE_NP, reason="Numpy not available")
+    @pytest.mark.parametrize("path", [EXPL_1_1_3F.path, EXPL_1_1_3F_NONALIGNED.path])
+    def test_compress_arr_1bit(self, path):
+        """Test compressing using pixel data from an arr."""
+        ds = dcmread(path)
+        assert hasattr(ds, "file_meta")
+        ref = ds.pixel_array
+        compress(ds, DeflatedImageFrameCompression, encoding_plugin="pydicom")
```

Comment:
You haven't passed the numpy array to `compress()`

Reply:
Addressed in [e9ea3de](https://github.com/pydicom/pydicom/pull/2216/commits/e9ea3deb538f6786f5a80e0e826d7aaf7a40678c)